### PR TITLE
added getting release and git clone to readme

### PR DESCRIPTION
**Pull request by puja108**
Seems we had forgotten this

RFR @teemow @marians

### CODE REVIEW

**Comment by puja108** [comment]
I used the github latest release link cause I couldn't find onsho on s3, should be ok, right?

File: README.md [hunk 1]
```diff
@@ -8,11 +8,9 @@ Manage QEMU VMs that will be provisioned by [Mayu](https://github.com/giantswarm
 
 ## Getting Onsho
 
-Three ways to get the project:
+Download the latest release from here: https://github.com/giantswarm/onsho/releases/latest
 
-- Install/Download a release
-- Docker image for a release (autobuild)
-- Build (two options: local or in docker)
+Clone the latest git repository version from here: git@github.com:giantswarm/onsho.git
```
Comment (zeisss) [inline README.md]: lets use a https url here, some people still have no git setup for github.

**Comment by marians** [comment]
Seems okay, but content-wise I can't really judge if that makes sense there.

File: README.md (hunk 1 above)
Comment (puja108) [inline README.md]: hmm, you mean `https://github.com/giantswarm/onsho.git` instead of the ssh one?

Comment (zeisss) [inline README.md]: yes

Comment (marians) [inline README.md]: +1 for https. Works more universally.

**Pull request merged**